templates/github/.ci/ansible/settings.py.j2.copy

```diff
@@ -26,127 +26,3 @@ API_ROOT = {{ api_root | repr }}
 {% endfor %}
 {% endif %}
 
-{# =======================================
-Macros for legacy and new storage settings
-========================================== -#}
-
-{%- macro s3_settings(legacy) -%}
-    {%- if legacy %}
-DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
-AWS_ACCESS_KEY_ID = "{{ minio_access_key }}"
-AWS_SECRET_ACCESS_KEY = "{{ minio_secret_key }}"
-AWS_S3_REGION_NAME = "eu-central-1"
-AWS_S3_ADDRESSING_STYLE = "path"
-AWS_S3_SIGNATURE_VERSION = "s3v4"
-AWS_STORAGE_BUCKET_NAME = "pulp3"
-AWS_S3_ENDPOINT_URL = "http://minio:9000"
-AWS_DEFAULT_ACL = "@none None"
-    {%- else %}
-STORAGES = {
-    "default": {
-        "BACKEND": "storages.backends.s3boto3.S3Boto3Storage",
-        "OPTIONS": {
-            "access_key": "{{ minio_access_key }}",
-            "secret_key": "{{ minio_secret_key }}",
-            "region_name": "eu-central-1",
-            "addressing_style": "path",
-            "signature_version": "s3v4",
-            "bucket_name": "pulp3",
-            "endpoint_url": "http://minio:9000",
-            "default_acl": "@none None",
-        },
-    },
-    "staticfiles": {
-      "BACKEND": "django.contrib.staticfiles.storage.StaticFilesStorage",
-    },
-}
-    {%- endif %}
-{%- endmacro -%}
-
-{%- macro azure_settings(legacy) -%}
-    {%- if legacy %}
-DEFAULT_FILE_STORAGE = "storages.backends.azure_storage.AzureStorage"
-AZURE_ACCOUNT_KEY = "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=="
-AZURE_ACCOUNT_NAME = "devstoreaccount1"
-AZURE_CONTAINER =  "pulp-test"
-AZURE_LOCATION = "pulp3"
-AZURE_OVERWRITE_FILES = True
-AZURE_URL_EXPIRATION_SECS = 120
-AZURE_CONNECTION_STRING = 'DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://ci-azurite:10000/devstoreaccount1;'
```

Comment:
Do you want to "move" this information into the default template values?
(I know it only helps a newly templated plugin, but it also serves as some sort of documentation.)